### PR TITLE
capability to run ansible from user

### PR DESCRIPTION
**Pull request by miteshget**
##### SUMMARY

ssh keys and configs added to {{student_user}} home

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
config: three-tier-app

### CODE REVIEW

File: ansible/configs/three-tier-app/pre_software.yml [hunk 1]
```diff
@@ -8,21 +9,61 @@
...
-  roles:
-    -  { role: "bastion", when: 'install_bastion' }
-    - role: bastion-student-user
-      when: install_student_user | bool
   tags:
     - step004
     - bastion_tasks
+  roles:
+    -  { role: "bastion", when: install_bastion | bool }
+    -  { role: bastion-student-user, when: install_student_user | bool }
```
Comment (fridim) [inline ansible/configs/three-tier-app/pre_software.yml:25]: i would use `include_role` when possible

as a task, not in the play definition

Comment (prakhar1985) [reply ansible/configs/three-tier-app/pre_software.yml:25]: Yes, You are right but I thought not to change any thing which is already exist instead add new stuff.  Any way will change to include_role.

Comment (miteshget) [reply ansible/configs/three-tier-app/pre_software.yml:25]: @prakhar1985 @fridim , I will change as per suggestion and push to git

Comment (miteshget) [reply ansible/configs/three-tier-app/pre_software.yml]: @fridim and @prakhar1985 I have updated the code as per suggestion.

**Comment by miteshget** [comment]
@fridim @prakhar1985 Please merge

**Pull request merged**